mava/utils/environments/pettingzoo_utils.py

```diff
@@ -131,6 +131,6 @@ def make_environment(
         raise Exception("Pettingzoo is not installed.")
 
     if concat_agent_id:
-        env = ConcatAgentIdToObservation(env)
+        environment = ConcatAgentIdToObservation(environment)  # type: ignore
```

Comment:
Thanks @RuanJohn . Okay cool I didn't know about the # type ignore flag. I actually added a fix to this in the frame staking PR but I had to define env as type Any to bypass the pre-commit type error.

Reply:
Thank you @ulricharmel, I try to use it very sparingly :sweat_smile: but in this case all my ideas for getting around the type checking made the code much less clean to read. The Any idea is also good.